test/decomposer/test_xzx_decomposer.py

```diff
@@ -54,3 +54,10 @@ def test_xzx_decomposer(decomposer: XZXDecomposer, gate: Gate, expected_result:
     decomposed_gate = decomposer.decompose(gate)
     check_gate_replacement(gate, decomposed_gate)
     assert decomposer.decompose(gate) == expected_result
+
+
+def test_axis_indexer():
+    xzx_decomp = XZXDecomposer()
+    missing_index = xzx_decomp._find_non_used_index([xzx_decomp.index_a, xzx_decomp.index_b])
```

Comment:
```suggestion
    missing_index = xzx_decomp._find_non_used_index()
```
If you accept the change, this should be changed